docs/en/using_starrocks/Using_HLL.md

```diff
@@ -59,7 +59,7 @@ This is the core idea of the HLL algorithm. Please refer to the [HyperLogLog pap
 ### How to use HyperLogLog
 
 1. To use HyperLogLog de-duplication, you need to set the target indicator column type to `HLL` and the aggregation function to `HLL_UNION` in the table creation statement.
-2. Currently, only the aggregation model supports HLL as indicator column type.
+2. Currently, only the aggregate table supports HLL as indicator column type.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/f33ef1ba-4585-40a6-abb0-033238ccc7d4)
-> the Aggregate table. Make sure to capitalize all table types (Primary Key, Aggregate, Duplicate Key, and Unique Key) in all docs involved.